src/Sylius/Bundle/ShopBundle/Resources/views/Product/Index/_sorting.html.twig

```diff
@@ -33,7 +33,7 @@
 
 <div class="ui right floated small header">
     <div class="content">
-        {{ 'sylius.ui.display'|trans }}
+        {{ 'sylius.ui.sort'|trans }}
```

Comment:
Is there such a message key as `sylius.ui.sort`? I do not see it in the UiBundle.